<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản Lý Công Văn Đi</string>
  </property>
  <property name="styleSheet">
   <string>
    QMainWindow {
     background-color: #f0f0f0;
    }
    QLabel {
     font-weight: bold;
     color: #2c3e50;
    }
    QLineEdit, QTextEdit, QDateEdit, QComboBox {
     border: 1px solid #bdc3c7;
     border-radius: 5px;
     padding: 5px;
     background-color: #ffffff;
    }
    QPushButton {
     background-color: #3498db;
     color: white;
     border-radius: 5px;
     padding: 5px 15px;
    }
    QPushButton:hover {
     background-color: #2980b9;
    }
    QTableWidget {
     border: 1px solid #bdc3c7;
     border-radius: 5px;
     background-color: #ffffff;
    }
    QTableWidget::item:selected {
     background-color: #3498db;
     color: white;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="styleSheet">
       <string>font-size: 18pt; font-weight: bold; color: #2980b9; margin: 10px;</string>
      </property>
      <property name="text">
       <string>QUẢN LÝ CÔNG VĂN ĐI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMaVanThu">
        <property name="text">
         <string>Công văn đi ID (*)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtcongvanid"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSoCongVan">
        <property name="text">
         <string>Số công văn (*)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtsocongvan"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblNoiDung">
        <property name="text">
         <string>Nội dung (*)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="txtnoidung"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblNguoiKy">
        <property name="text">
         <string>Người ký</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtnguoiky"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblNgayPhat">
        <property name="text">
         <string>Ngày phát (*)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="datengayphat"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblNoiNhan">
        <property name="text">
         <string>Nơi nhận (*)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblLoaiCongVan">
        <property name="text">
         <string>Loại công văn</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cbloaicongvan">
        <item>
         <property name="text">
          <string>Gửi đối tác</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gửi cơ quan nhà nước</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Khẩn cấp</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblTrangThai">
        <property name="text">
         <string>Trạng thái</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="cbtrangthai">
        <item>
         <property name="text">
          <string>Chưa gửi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Đang chờ duyệt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Đã gửi</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lblFileDinhKem">
        <property name="text">
         <string>Đường link đính kèm</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txtfiledinhkem"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblNoiNhan_2">
        <property name="text">
         <string>Phòng ban phụ trách (*)</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="btnChonFile">
        <property name="text">
         <string>Chọn file</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cbphongban">
        <item>
         <property name="text">
          <string>Phòng Hành Chính</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phòng Kế Toán</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phòng Nhân Sự</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phòng Kỹ Thuật</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phòng Kinh Doanh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phòng Tài Chính</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cbnoinhan">
        <item>
         <property name="text">
          <string>Sở Giáo Dục</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sở Y Tế</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bộ Giao Thông Vận Tải</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bộ Công An</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bộ Công Thương</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chính Phủ</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableCongVan">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnGhiDuLieu">
        <property name="text">
         <string>Ghi dữ liệu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHuy">
        <property name="text">
         <string>Hủy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
